### PR TITLE
Add github issue templates

### PR DESCRIPTION
**Pull request by yoshuawuyts**
This adds issue templates to better help guide customers to the resources they're looking for. You can find a live demo [here](https://github.com/yoshuawuyts/issue-forms-test/issues/new/choose). Thanks!

## Screenshots

New-Issue-·-yoshuawuyts-issue-forms-test

New-Issue-·-yoshuawuyts-test

New-Issue-·-yoshuawuyts-issue-forms-test (1)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/blank_issue.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+name: Blank Issue
+about: Create a blank issue.
```
Comment (kennykerr) [inline .github/ISSUE_TEMPLATE/blank_issue.md:3]:
```suggestion
about: Create a blank issue
```
Consistency with other subheads.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Windows API question
+    url: https://stackoverflow.com/questions/tagged/winapi
+    about: Please ask and answer questions about the Windows API on StackOverflow.
```
Comment (kennykerr) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: Please ask and answer questions about the Windows API on StackOverflow
```

**Comment by yoshuawuyts** [comment]
Implemented feedback from review!

**Review by kennykerr — APPROVED** [review]
Thanks!

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Windows API question
+    url: https://stackoverflow.com/questions/tagged/winapi
+    about: Please ask and answer questions about the Windows API on StackOverflow
```
Comment (riverar) [inline .github/ISSUE_TEMPLATE/config.yml:5]: Think there's a dedicated support staff for general Windows API development on Q&A, so should we at least try pushing folks that way first?

**Review by riverar — APPROVED** [review]
Looks great, just one question/nit.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (yoshuawuyts) [reply .github/ISSUE_TEMPLATE/config.yml:5]: Sounds good; will update!

**Comment by yoshuawuyts** [comment]
updated; think this is ready to merge!

**Review by kennykerr — APPROVED** [review]

**Pull request merged**